src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -144,8 +145,12 @@ export type Fiber = {
 
 };
 
+export type FiberDev = Fiber & {
```

Comment:
If I add it to the Fiber how can I convince Flow it exists even if I don't set it in DEV? Do the coercion when returning?